browser/brave_rewards/android/brave_rewards_native_worker.cc

```diff
@@ -651,16 +651,15 @@ void BraveRewardsNativeWorker::GetExternalWallet(
     JNIEnv* env,
     const base::android::JavaParamRef<jobject>& obj) {
   if (brave_rewards_service_) {
-    auto callback = base::Bind(
-        &BraveRewardsNativeWorker::OnGetExternalWallet,
-        base::Unretained(this));
-    brave_rewards_service_->GetUpholdWallet(callback);
+    brave_rewards_service_->GetExternalWallet(
+        base::Bind(&BraveRewardsNativeWorker::OnGetExternalWallet,
```

Comment:
Maybe we can make this a `base::BindOnce` while we're touching this?

Reply:
Fixed.